config/index.js

```diff
@@ -654,6 +654,12 @@ var conf = convict({
       default: /.+@mozilla\.com$/,
       env: 'SIGNIN_CONFIRMATION_FORCE_EMAIL_REGEX'
     },
+    skipForEmailAddresses: {
+      doc: 'Comma separated list of email addresses that will always skip any non TOTP sign-in confirmation',
```

Comment:
TODO: @vbudhram will double-check how this plays nicely with convict.

Reply:
Confirmed that they are comma separated, ref https://github.com/mozilla-services/cloudops-deployment/blob/1afc24510fe98ce034b7308aa186c72e00fa6c8c/projects/fxa/puppet/modules/fxa_auth/manifests/init.pp#L141 which uses the SMS country codes array.